src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const collectionObjects = [];
+  let objectStep = { ...state };
```

Comment:
```suggestion
  let clonedState = { ...state };
```
as a variant. Not really clear what `objectStep` should mean

Reply:
Give good variable`s name it`s art!)